src/Literal.ts

```diff
@@ -13,15 +13,14 @@ interface Literal<T extends LiteralBase = LiteralBase> extends Runtype.Common<T>
 	value: T
 }
 
-/**
- * Be aware of an Array of Symbols `[Symbol()]` which would throw "TypeError: Cannot convert a Symbol value to a string"
- */
-const literal = (value: unknown) =>
+const literal = (value: unknown): string =>
 	Array.isArray(value)
-		? globalThis.String(value.map(globalThis.String))
-		: typeof value === "bigint"
-			? globalThis.String(value) + "n"
-			: globalThis.String(value)
+		? globalThis.String(value.map(literal))
```

Comment:
Recursing `literal` will handle null prototype objects, BigInt formatting, etc. within an array.

Are there any potential issues here? I'm just cautious because it wasn't implemented this way before 🤔

Reply:
Aha, CI says yes 🙂

Reply:
I can think of two approaches here:

1. Just `catch` the error when it crops up and return some placeholder string. Hopefully still be preferable over throwing a `TypeError`.
2. Stop unpacking a nested array after a certain depth. `literal` currently flattens the array so I'm not sure how valuable its output actually is in practice.